design/ceph-csi-operator.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Integrate cephcsi operator with Rook
+target-version: release-1.15
+---
+
+# Integrate cephcsi operator with Rook
+
+## Overview
+
+To integrate the new ceph-csi operator with Rook and remove the existing csi deployment related code, RBAC's and dead code in later releases. Also before moving forward please read the ceph-csi-operator design doc [here](https://github.com/ceph/ceph-csi-operator/blob/main/docs/design/operator.md).
+
+## Rook Deploying Ceph-CSI-Operator
+
+There are two scenarios rook needs to manage while deploying of the new csi-operator.
+
+1. Deployment in new cluster i.e [Fresh deployment](#fresh-deployment)
+    1. Manifest Deployment
+    2. Helm Deployment
```

Comment:
I think it would also be worth noting here (and in docs) that multus users should not use this until it's stable or until holder pods are removed.

IMO, it will be simplest to tell users not to use this when multus is enabled. That is a clear instruction. We can remove the note once holder support is removed entirely.

Reply:
If the holder pod is enabled, how about if Rook ignores the setting(s) to configure the CSI operator?

Reply:
Yes, I'll skip deploying CSI-operator when multus is true also added note in design